### PR TITLE
Add preliminary support for Pico 6000A/B/C/D

### PR DESCRIPTION
**Pull request by TiltMeSenpai**
Tl;dr, versions of the Picoscope 6000 series before the 6000E's use a different API, similar to the 3000a API currently supported.
WIP PR to add support for these scopes.

Known Issues:
- [ ] I still haven't done any type of formatting passes or anything
- [ ] There's a couple low-effort early exits for AWG stuff that should be removed or cleaned up
- [ ] The Pico 6404D (at minimum) has an AWG that should get similar support to the 6404E
- [ ] Odds are this PR will conflict with other PRs that have been WIP for longer (e.g. support for the non-a API 3000 series scopes)

Known Issues possibly related to the glscope driver, not the picoscope bridge
- [x] glscopeclient crashes when trying to view/edit timebase settings (clicking on the timescale at the top)
- [ ] glscopeclient thinks the 6404D has support for logic analyzer pods (it does not)

### CODE REVIEW

**Comment by bvernoux** [comment]
It will be great to wait the merge of https://github.com/glscopeclient/scopehal-pico-bridge/pull/19 (and or to be synchronized with it) before to merge that PR

**Comment by TiltMeSenpai** [comment]
Yep, at minimum I'll wait for #19 to be merged before marking this out of WIP

**Comment by bvernoux** [comment]
https://github.com/glscopeclient/scopehal-pico-bridge/pull/19 is merged now so you can update your code with it

**Comment by TiltMeSenpai** [comment]
It's been like a year and updating would be a mess, trying again, apologies for the chaos
